### ISSUE TITLE
Build fails with GOOS=linux GOARCH=386 due to int overflow

### ISSUE THREAD

**Issue by astj** [opencontainers/runc#1818]
In fresh golang environments (in docker container):
```
$ docker run --rm -it golang bash
root@0b225ff49f25:/go# go get github.com/opencontainers/runc
root@0b225ff49f25:/go# go version
go version go1.10.3 linux/amd64

(snip)

root@0b225ff49f25:/go/src/github.com/opencontainers/runc# make BUILDTAGS="" GOARCH=386
go build -buildmode=pie  -ldflags "-X main.gitCommit="dd56ece8236d6d9e5bed4ea0c31fe53c7b873ff4" -X main.version=1.0.0-rc5+dev " -tags "" -o runc .
go build github.com/opencontainers/runc/libcontainer/nsenter: build constraints exclude all Go files in /go/src/github.com/opencontainers/runc/libcontainer/nsenter
# github.com/opencontainers/runc/libcontainer/system
libcontainer/system/linux.go:119:89: constant 4294967295 overflows int
Makefile:32: recipe for target 'runc' failed
```

It looks like this is caused by #1688. `IDMap.Count` is defined as int, and in 32bit environment 4294967295 is too large for int.

(In fact I don't use runc directly, but uses opencontainers/runc/libcontainer from other libraries.)

**Comment by cyphar** [comment]
This will be fixed by #1816.

**Comment by astj** [comment]
Thank you!
